sdk/src/test/java/com/horizen/transaction/RegularTransactionTest.java

```diff
@@ -205,21 +244,31 @@ public void forgerBoxTest() {
 
         // Test 2: Create new transaction with regular boxes and forger boxes
         to.clear();
-        to.add(new RegularBoxData(getPrivateKey25519().publicImage(), 30L));
-        to.add(new RegularBoxData(getPrivateKey25519().publicImage(), 50L));
+        to.add(new ZenBoxData(getPrivateKey25519().publicImage(), 30L));
+        to.add(new ZenBoxData(getPrivateKey25519().publicImage(), 50L));
         to.add(new ForgerBoxData(getPrivateKey25519().publicImage(), 30L, getPrivateKey25519().publicImage(), getVRFPublicKey()));
         to.add(new ForgerBoxData(getPrivateKey25519().publicImage(), 10L, getPrivateKey25519().publicImage(), getVRFPublicKey()));
 
-        RegularTransaction tx2 = RegularTransaction.create(from, to, fee, timestamp);
+        RegularTransaction tx2 = RegularTransaction.create(from, to, fee);
 
         List<NoncedBox<Proposition>> tx2NewBoxes = tx2.newBoxes();
-        assertTrue("Transaction must be semantically valid.", tx2.semanticValidity());
+
+        try {
+            tx2.semanticValidity();
+            isValid = true;
+        } catch (TransactionSemanticValidityException e) {
+            isValid = false;
+        } catch (Exception e) {
+            fail("TransactionSemanticValidityException type expected.");
+        }
+        assertTrue("Transaction expected to be semantically valid.", isValid);
+
         assertEquals("Count of new boxes must be the same as count of regular boxes and forger boxes.",
                 to.size(), tx2NewBoxes.size());
         for(NoncedBox box : tx2NewBoxes ) {
-            if (box instanceof RegularBox)
+            if (box instanceof ZenBox)
                 assertTrue("Transaction must contain new box for specified regular boxes data.",
-                        to.contains(new RegularBoxData((PublicKey25519Proposition)box.proposition(), box.value())));
+                        to.contains(new ZenBoxData((PublicKey25519Proposition)box.proposition(), box.value())));
```

Comment:
Update box name in assertion message(lines 266, 270).
Also update box name in fail message on line 275.